### DISCUSSION TITLE
Energy calculation not proper

### DISCUSSION THREAD

**Discussion by rohanpatil-rc** [arendst/Tasmota#22655] [Q&A] [answered]
I Have [Wipro 16A Power Monitoring Plug](https://templates.blakadder.com/wipro_DS11160.html)
Monitoring refrigerator it has BL0937 with tuya chip which i replace with esp8266(tasmota 14.3.0). the reading for 14dec are weird 
screenshots  to explain the issues i have install it 2days ago and those reading are perfect. I am new to HA and tasmota may be i am doing something wrong help
![today](https://github.com/user-attachments/assets/e7574528-4029-4b35-8d00-edc8178befe3)
![Screenshot 2024-12-15 125106](https://github.com/user-attachments/assets/273ce53a-8049-4d4b-98f8-0aaff726a181)
![total](https://github.com/user-attachments/assets/be06c622-18f3-4f44-94ac-52c399708394)

**Comment by sfromis** [comment]
I see nothing "weird", just signs that your device experiences loss of input power. Tasmota does not wear down flash by continuously writing persistent data.

**Comment by rohanpatil-rc** [comment, marked as answer]
> I see nothing "weird", just signs that your device experiences loss of input power. Tasmota does not wear down flash by continuously writing persistent data.

ok i will try with different appliance and resolder the esp8266 connection.
i started noticing that it goes offline for few seconds and comes back

**Reply by rohanpatil-rc** [reply]
found the problem was the ground wire which got pinch cause esp to shutdown randomly
sorry @Jason2866 for posting the problem in the issues section.
i am new to iot,HA & tasmota community

**Reply by Jason2866** [reply]
Great you find the reason. No worries about posting in issue. We all started some day :-)
